components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ApplicationBasicInfo.java

```diff
@@ -179,23 +180,43 @@ public void setAppOwner(User appOwner) {
     }
 
     /**
-     * Get inbound auth key.
+     * Get oAuth2 client ID.
```

Comment:
It is `OAuth2` not `oAtuh2`, please check the other places as well

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/4432